### ISSUE TITLE
.gz conflict with compressed core files

### ISSUE THREAD

**Issue by kraemer-igroup** [lochmueller/staticfilecache#402]
Hi there,

I noticed an issue with TYPO3 v12 and the changed file extension of compressed files from .gzip to .gz (https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/12.0/Breaking-93182-ChangedFileExtensionForGzipCompressedFiles.html).

The corresponding .htaccess rules get overridden by the suggested staticfilecache configuration, which forces the type text/html for all .gz files:
```
<FilesMatch "\.gz$">
	ForceType text/html
	# AddType "text/html" .gz
</FilesMatch>
```
A solution could be, that the suggested staticfilecache configuration also provides correct file types for `.js.gz` and `.css.gz`:
```
<FilesMatch "\.gz$">
	ForceType text/html
	# AddType "text/html" .gz
</FilesMatch>
<FilesMatch "\.js\.gz$">
	ForceType text/javascript
	# AddType "text/javascript" .gz
</FilesMatch>
<FilesMatch "\.css\.gz$">
	ForceType text/css
	# AddType "text/css" .gz
</FilesMatch>
```

**Comment by kraemer-igroup** [comment]
Hey @lochmueller, I just stumbled upon this in a current v13 project. I noticed, that the `\.js\.gz$` and `\.css\.gz$` rules are duplicated, which may be confusing.
I guess, that the fix for #399 got inserted in the wrong place and those changes got overridden by the more general `\.gz$` rule? So maybe you could replace this:
```
# Set proper content type gzipped html
<FilesMatch "\.gzip$">
   ForceType text/html
   # AddType "text/html" .gzip
</FilesMatch>
<FilesMatch "\.js\.gzip$">
   ForceType text/javascript
   # AddType "text/javascript" .gzip
</FilesMatch>
<FilesMatch "\.js\.gz$">
   ForceType text/javascript
   # AddType "text/javascript" .gzip
</FilesMatch>
<FilesMatch "\.css\.gzip$">
   ForceType text/css
   # AddType "text/css" .gzip
</FilesMatch>
<FilesMatch "\.css\.gz$">
   ForceType text/css
   # AddType "text/css" .gzip
</FilesMatch>
<FilesMatch "\.xml\.gzip$">
   ForceType text/xml
   # AddType "text/xml" .gzip
</FilesMatch>
```

with:
```
# Set proper content type gzipped html
<FilesMatch "\.gzip$">
   ForceType text/html
   # AddType "text/html" .gzip
</FilesMatch>
<FilesMatch "\.js\.gzip$">
   ForceType text/javascript
   # AddType "text/javascript" .gzip
</FilesMatch>
<FilesMatch "\.css\.gzip$">
   ForceType text/css
   # AddType "text/css" .gzip
</FilesMatch>
<FilesMatch "\.xml\.gzip$">
   ForceType text/xml
   # AddType "text/xml" .gzip
</FilesMatch>
```

in the current `Htaccess.rst`, because the right place is below the general `\.gz$` rule, where the duplicates already are?
But I'm no .htaccess expert, maybe it has to be the way it is.

**Comment by kraemer-igroup** [comment]
Hey @lochmueller 
everything works fine, I just noticed duplicated rules in
https://github.com/lochmueller/staticfilecache/blob/main/Documentation/Configuration/Htaccess.rst?plain=1
(lines 150-153 duplicated by 174-177 and lines 158-161 duplicated by 178-181)

Maybe with this screenshot it is more clearly:
![Image](https://github.com/user-attachments/assets/8511dd3c-cf64-4a2e-b6cb-740cd03c0f14)

I wanted to cleanup my grown .htaccess file and was confused about the duplicates and needed a little time to determine, which of them could be removed (the latter ones must stay).

Just wanted to give you a note about this :-)
